### DISCUSSION TITLE
How to save yalmip object

### DISCUSSION THREAD

**Discussion by wangwendi0705** [yalmip/YALMIP#1219] [Problems using YALMIP] [answered]
Hi Johan,
When I use the solvemp command to create a EMPC object，I want to save the ZPWF (the optimal decision variable as a pfw function) in order to use it in other place. However, it indicates that "YALMIP objects cannot be saved in binary format". Is there any solutions?

**Comment by johanlofberg** [comment, marked as answer]
No, you will have to save the MPT structure and work via that
